src/rtc_pl031.rs

```diff
@@ -147,8 +207,11 @@ impl RTC {
 
         match offset {
             RTCMR => {
-                // Set the match register, though this is not currently used.
+                // Set the match register.
+                // Since this is not currently used, we call missed_write()
+                // from the rtc_events struct.
                 // TODO: Implement the match register functionality.
+                self.rtc_events.missed_write();
```

Comment:
Just wondering, is this metric actually useful? Since we are mentioning in the README that the match register is not implemented, isn't this "missed write" kind of expected behavior? I read the other related thread, but I'm still not so sure about the value of this metric (and of `missed_read`) :-?
If we want to add a metric for the specific alarm thing or for unimplemented things, maybe we can somehow rename these metric(s). `missed` is pretty confusing.